src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: $calendar-padding;
+  width: $calendar-width;
+
```

To center calendar add this one
```suggestion
  width: $calendar-width;
  margin: 0 auto;
```